jdaviz/configs/cubeviz/helper.py

```diff
@@ -120,26 +143,21 @@ def specviz(self):
             self._specviz = Specviz(app=self.app)
         return self._specviz
 
-    def get_data(self, data_label=None, spatial_subset=None, spectral_subset=None, function=None,
```

Comment:
We don't have to deprecate `function` here? Can use [deprecated_renamed_argument](https://docs.astropy.org/en/latest/api/astropy.utils.decorators.deprecated_renamed_argument.html#astropy.utils.decorators.deprecated_renamed_argument).

Reply:
This is a large reason why we're bumping to 4.0 once we merge this PR (this also makes a few breaking changes to plugins).  If there is a way to decorate and note the removal without having to temporarily support the old kwargs, I have no objections, but keeping the old code around with a warning proved to be messy and inconsistent with the "new way".

Reply:
yes, for sure, this PR will require a major change log entry (probably many of the entries in the PR description will need to be included somehow).